src/types.rs

```diff
@@ -22,6 +28,12 @@ use crate::CompressionMethod;
 #[cfg(feature = "time")]
 use time::{error::ComponentRange, Date, Month, OffsetDateTime, PrimitiveDateTime, Time};
 
+pub(crate) struct ZipRawValues {
```

Comment:
Done! Also added accessor methods for these to `ZipFileData`.

Reply:
@Pr0methean: actually, I'd like your thoughts on this. This produces what feels like an unnecessary breaking API change, from `ZipFileData::compressed_size` (a field) to `ZipFileData::compressed_size()` (a method) e.g. https://github.com/zip-rs/zip2/pull/93/files#diff-8e31462ea4b11fdadc15fd98e895c1902595d9d34947c58eecca1125a5e6224dL559-R557. I don't think that's necessarily a problem, but it seems annoying. Would you prefer that I revert the creation of this new `ZipRawValues` struct and retain the previous API? I think I'd prefer that, personally, but I could also see an argument to keep it this way.

The `ZipRawValues` struct was only intended as a way to add type safety in our internal implementation, so it's more difficult to pass around the wrong length fields to the `ZipFileData::initialize_local_block()` method. I still think it's useful for that purpose, but I don't think it makes sense to change the external API for this. Would you accept repeating these fields in the internal `ZipRawValues` struct and the external `ZipFileData` struct to avoid this API change? Or would you prefer I simply remove `ZipRawValues` entirely (which makes it slightly easier to accidentally mix up the `compressed_size` and `uncompressed_size` variables in our implementation code)? Let me know what you think.